apps/asap-server/src/handlers/api-handler.ts

```diff
@@ -40,3 +41,13 @@ export const apiHandler = serverlessHttp(app, {
     logger.withRequest(event, context);
   },
 });
+
+Sentry.AWSLambda.init({
+  dsn: "https://c61a36e015ef481bb29e0bd2a9f84aa3@o850903.ingest.sentry.io/5817859",
```

Comment:
Does this mean all environments including localhost will report errors to the same place?

Reply:
perhaps i should have made ti clear that this was a POC